third_party/android_deps/libs/com_android_support_support_annotations/README.chromium

```diff
@@ -0,0 +1,19 @@
+Name: Android Support Library Annotations
+Short Name: support-annotations
+URL: https://dl.google.com/dl/android/maven2/com/android/support/support-annotations/28.0.0/support-annotations-28.0.0.jar
+Version: 28.0.0
+Update Mechanism: Manual
+License: Apache-2.0
+License File: LICENSE
```

Comment:
With this, it's no longer necessary to have a copy of the license in the `LICENSE` file.
```suggestion
License File: /brave/common/licenses/Apache-2.0
```

Reply:
Updated at https://github.com/brave/brave-core/pull/32111/commits/ad1a7617b175f93f0074696037dfe95cc0fe385a and https://github.com/brave/brave-core/pull/32111/commits/bd58664b9bd323f91741c310163d12a608f4e604